src/__tests__/index.spec.ts

```diff
@@ -158,6 +158,38 @@ describe("type-coverage", () => {
     },
   ],
 }
+`);
+  });
+
+  it("should handle atLeast option", async () => {
+    codechecksMock.isPr.mockReturnValue(true);
+    typeCoverageMock.mockReturnValue({
+      correctCount: 4,
+      totalCount: 10,
+      anys: [],
+      program: undefined as any,
+    });
+
+    await typeCoverageWatcher({ tsconfigPath: "./tsconfig.json", atLeast: 50 });
+    expect(codechecks.report).toMatchInlineSnapshot(`
+[MockFunction] {
+  "calls": Array [
+    Array [
+      Object {
+        "longDescription": "New untyped symbols: 0",
```

Comment:
I'm not sure what we should add to the long description in this case, any thoughts?

Reply:
I think it's fine to begin with. If you use `atLeast` option please provide feedback later :)